crypto/hpke/hpke.c

```diff
@@ -1,7 +1,8 @@
 /*
- * Copyright 2022-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2000-2023 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Why the year change?

Reply:
Done. Addressed.

It was copy paste mistake. Thanks!